CONTRIBUTING.md

```diff
@@ -1,122 +1,172 @@
+* A new **branch** of code to work on.
 
-## Installing Prerequisites
+#### Java
 
-Before you get started, you need to set yourself up with an environment in which to develop Mule.  Your dev environment needs three things: a Java SDK, a recent version of Maven, an integration development environment (IDE), and new branch of code to work on.
+1. If you are working with *Windows* or **Linux**, install one of the following [Java Development Kits](http://www.oracle.com/technetwork/java/javase/downloads/index.html) on your local drive.
+2. If you are working on a **Mac**, simply confirm that the JDK shipped with your Mac OS X is _Java SE Development Kit 7 (also known as Java SE)_ or newer using the command
+`java -version`. Then skip to step 4 below.
+3. Create an environment variable called `JAVA_HOME`, setting it to the directory in which you installed the JDK.
+4. Update the `PATH` environment variable so that it includes the path to JDK binaries. Add the following to the `PATH` variable:
 
-### JDK
+    On **Windows**
+    ```
+    %JAVA_HOME%/bin
+    ```
 
-1. If you are working with **Windows** or **Linux**, install one of the following [Java Development Kits](http://www.oracle.com/technetwork/java/javase/downloads/index.html) on your local drive. If you are working on a **Mac**, simply confirm that the JDK shipped with your Mac OS X is *Java SE Development Kit 7 (also known as Java SE 7u80)* or newer using the command `java -version`, then skip to step 4 below: 
-2. Create an environment variable called `JAVA_HOME`, setting it to the directory in which you installed the JDK. 
-3. Update the PATH environment variable so that it includes the path to JDK binaries. Add the following line to the PATH variable:
-    - Windows: `%JAVA_HOME%/bin`
-    - Linux or Mac OS X: `$JAVA_HOME/bin`
-4. If you are using a Mac OS X, examine the contents of the `$JAVA_HOME/jre/lib/` security directory to confirm that the following two files are present:  
+    On **Linux or Mac OS X**
+    ```
+    $JAVA_HOME/bin
+    ```
+
+5. If you are using a Mac OS X, examine the contents of the `$JAVA_HOME/jre/lib/` security directory to confirm that the following two files are present:  
     - local_policy.jar
-    - US_export_policy.jar  
-These two files prevent any problems regarding cryptography. If not present, download the [Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files 6.0](http://www.oracle.com/technetwork/java/javase/downloads/jce-6-download-429243.html), then copy the files into the security directory identified above.
+    - US_export_policy.jar 
+
+   These two files prevent any problems regarding cryptography. If not present, download the [Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files 6.0](http://www.oracle.com/technetwork/java/javase/downloads/jce-6-download-429243.html), then copy the files into the security directory identified above.
```

Comment:
Te be consistent, we should recommend JCE 8.0 for JDK8: http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html